anasymod/emu/xsct_emu.py

```diff
@@ -30,7 +30,10 @@ def bit_path(self):
 
     @property
     def tcl_path(self):
-        return self.impl_dir / 'ps7_init.tcl'
+        if self.target.prj_cfg.board.is_ultrascale:
+            return self.impl_dir / 'psu_init.tcl'
```

Comment:
``psu_init.tcl`` is not used for ZCU106 because only the FSBL initialization works for that board.  Perhaps in the future we could test out whether there is any benefit to using TCL initialization for other UltraScale+ boards.